public/index.html

```diff
@@ -115,6 +132,50 @@ <h2>PayJoin here to open these channels</h2>
 				channels.removeChild(channels.lastElementChild);
 			}
 
+			async function get_recommended_channels(){
+				document.getElementsByClassName('recommendation-button-wrapper')[0].classList.add('invisible')
```

Comment:
When I used this at first it was confusing because the query takes time and the control just disappears. Would adding "Loading..." in liu of some animation fix that?

Reply:
Yeah thats not a bad suggestion